pact.cabal

```diff
@@ -139,37 +145,56 @@ library
                     Pact.Analyze.Types.Types
                     Pact.Analyze.Util
                     Pact.Analyze.Remote.Server
-                    Pact.ApiReq
                     Pact.Bench
-                    Pact.Coverage
-                    Pact.Coverage.Report
-                    Pact.Docgen
-                    Pact.Interpreter
                     Pact.Main
-                    Pact.MockDb
-                    Pact.Persist.MockPersist
-                    Pact.Persist.SQLite
-                    Pact.PersistPactDb.Regression
                     Pact.ReplTools
                     Pact.Server.ApiServer
                     Pact.Server.History.Persistence
                     Pact.Server.History.Service
                     Pact.Server.History.Types
-                    Pact.Server.PactService
                     Pact.Server.Server
                     Pact.Server.Test
-                    Pact.Types.Crypto
-                    Pact.Types.ECDSA
-                    Pact.Types.Server
-                    Pact.Types.SQLite
                     Pact.GasModel.GasModel
                     Pact.GasModel.Types
                     Pact.GasModel.Utils
                     Pact.GasModel.GasTests
 
+    build-depends:
+                  algebraic-graphs >= 0.2 && < 0.6
+                , cassava >= 0.5
+                , constraints
+                , fast-logger
+                , haskeline >= 0.7.3 && < 0.9
+                , mmorph >= 1.1 && < 1.2
+                , neat-interpolation >= 0.4
+                , optparse-applicative >= 0.12.1.0 && < 0.16
+                , sbv >= 8.8
+                , wai-cors
+                , warp
+
+
+    cpp-options: -DBUILD_TOOL
```

Comment:
This must also be set for the other components below. Otherwise running those causes an error.